esphome/components/pcd8544/display.py

```diff
@@ -33,6 +34,9 @@ def to_code(config):
     reset = yield cg.gpio_pin_expression(config[CONF_RESET_PIN])
     cg.add(var.set_reset_pin(reset))
 
+    if CONF_CONTRAST in config:
+        cg.add(var.set_contrast(config[CONF_CONTRAST]))
+
```

Comment:
If you have a default, you don't need to check
```suggestion
    cg.add(var.set_contrast(config[CONF_CONTRAST]))

```